master/internal/api_user_intg_test.go

```diff
@@ -625,6 +625,26 @@ func TestAuthzPostUser(t *testing.T) {
 	require.Equal(t, expectedErr.Error(), err.Error())
 }
 
+func TestAuthzPostUserDuplicate(t *testing.T) {
```

Comment:
this is just a drive by comment

We can just use 
```diff
+	api, curUser, ctx := setupAPITest(t, nil)
-	api, authzUsers, curUser, ctx := setupUserAuthzTest(t, nil)
```
 if we aren't testing rbac specifically and then shouldn't need to worry about any mocks

Reply:
I think we do need to mock the api/authzUsers for the postUser call (breaks otherwise) -- but I can remove 'curUser'

Reply:
Did you try it with?
```
api, curUser, ctx := setupAPITest(t, nil)
```

setupAPITest will make it use OSS authentication and shouldn't have any mocks?